lib/cpp/test/OpenSSLManualInitTest.cpp

```diff
@@ -67,6 +67,22 @@ void test_openssl_availability() {
   openssl_cleanup();
 }
 
```

Comment:
Frankly speaking, I'm not a Boost.Test expert. I had to Google a lot to find a way that could build and run tests successfully. Please apply any change that could clean the test code, and make it work in any case. My patch could not be the correct way to fix it (even if I can ensure you that it works fine, and all tests are passed).